janus_server/src/aggregator.rs

```diff
@@ -2727,6 +2745,8 @@ mod tests {
     use uuid::Uuid;
     use warp::{cors::CorsForbidden, reply::Reply, Rejection};
 
+    const DUMMY_VERIFY_KEY_LENGTH: usize = dummy_vdaf::Vdaf::VERIFY_KEY_LENGTH;
```

Comment:
this should be `#[cfg(test)]`, I think

Reply:
It's in `mod tests` so it should already be restricted to test builds.